src/classes/pilot/Pilot.ts

```diff
@@ -678,15 +678,15 @@ class Pilot
   public static Deserialize(pilotData: PilotData): Pilot {
     const p = new Pilot()
     try {
-      p.Update(pilotData)
+      p.Update(pilotData,true)
       p.SaveController.SetLoaded()
       return p
     } catch (err) {
       console.error(err)
     }
   }
 
-  public Update(data: PilotData, sync?: boolean): void {
+  public Update(data: PilotData, deserialize?: boolean): void {
```

Comment:
As Update is part of the ICloudSyncable API, we now assume that calls to update are sync operations and have an optional parameter to let us know if we are deserializing instead.